.github/workflows/dependabot_update_rbis.yml

```diff
@@ -0,0 +1,53 @@
+name: Update gem RBIs on Dependabot PRs
+
+on: workflow_call
+
+permissions:
+  contents: write
+  pull-requests: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  update-gem-rbis:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    if: ${{ github.actor == '"dependabot[bot]"' }}
+    steps:
+      - name: Checkout PR branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: ruby/setup-ruby@v1
+        id: setup
+        with:
+          bundler-cache: true
+          ruby-version: 3.2
+      - name: Ruby setup error details
+        if: ${{ failure() && steps.setup.conclusion == 'failure' }}
+        run: |
+          echo "A failure during the Ruby setup step"
+          exit 1
+      - name: Update gem RBIs
+        run: |
+          bundle exec tapioca gem
```

Comment:
```suggestion
          bin/tapioca gem
```
For consistency

Reply:
This comment was resolved, but the code wasn't updated.

Reply:
Fixed (I committed on GitHub.com but force pushed before pulling).